dfastmi/io/ConfigFileOperations.py

```diff
@@ -205,6 +206,68 @@ def load_configuration_file(filename: str) -> configparser.ConfigParser:
         rootdir = os.path.dirname(filename)
         return ConfigFileOperations._config_to_absolute_paths(rootdir, config)
 
+    @staticmethod
+    def _config_case_check(
+        config: configparser.ConfigParser,
+    ) -> configparser.ConfigParser:
+        config = ConfigFileOperations._config_case_check_section(config, "General")
+        for key in [
```

Comment:
Make use of the generalconfig class which uses pydantic to do this check automatically.

Reply:
New issue to be created.

Reply:
https://issuetracker.deltares.nl/browse/DFAST-216

Reply:
Discussed with Bert.